runtime/moonbase/tests/integration_test.rs

```diff
@@ -1412,3 +1414,126 @@ fn points_precompile_round_too_big_error() {
 		);
 	})
 }
+
+#[cfg(test)]
+mod multiplier_tests {
+	use super::*;
+	use frame_support::{
+		parameter_types,
+		weights::{DispatchClass, Weight},
+	};
+	use sp_core::H256;
+	use sp_runtime::{
+		testing::Header,
+		traits::{BlakeTwo256, Convert, IdentityLookup, One},
+		Perbill,
+	};
+
+	type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Runtime>;
+	type Block = frame_system::mocking::MockBlock<Runtime>;
+
+	frame_support::construct_runtime!(
+		pub enum Runtime where
+			Block = Block,
+			NodeBlock = Block,
+			UncheckedExtrinsic = UncheckedExtrinsic,
+		{
+			System: frame_system::{Pallet, Call, Config, Storage, Event<T>}
```

Comment:
If we move the test to this file, it'd be more consistent to test the actual runtime instead of this mock runtime with the feature enabled. Was there difficulty testing it using the actual runtime and, if so, what was the problem?

Reply:
WIP: f1c005d

They're not using a mocked runtime, but they are still using `TestExternalities` for something to do with weight that I'll have to wrap my head around.